File: StudyBuddy.Model/FcmToken.cs
using System;

namespace StudyBuddy.Model
{
    public class FcmToken
    {
        public int ID { get; set; }
        public string Token { get; set; }
        public int UserID { get; set; }
        public DateTime Created { get; set; } = DateTime.Now.Date;
        public DateTime LastSeen { get; set; } = DateTime.Now.Date;
    }

    public class FcmTokenSaveDto
    {
        public string Token { get; set; }

        public FcmToken ToFcmToken()
        {
            return new FcmToken
            {
                Token = Token
            };
        }
    }
}

File: StudyBuddy.Backend/src/app/model/fcmToken.ts

export class FcmToken {
    public id: number;
    public token: string;
    public userID: number;
    created: string;
    lastSeen: string;

    static fromApi(result):FcmToken {
        let obj = new FcmToken();
        obj.id = result["id"];
        obj.token = result["token"];
        obj.userID = result["userID"];
        obj.created = result["created"];
        obj.lastSeen = result["lastSeen"].split('T')[0];
        return obj;
    }
}